### DISCUSSION TITLE
OpenVPN Connect Localization

### DISCUSSION THREAD

**Discussion by DevDef** [OpenVPN/openvpn3#331] [General]
Hello,

I’m really disapointed that OpenVPN Connect is not localized. This is for end users, and it has to be the simplest as possible.

**Comment by schwabe** [comment]
Wrong issue tracker. Support for OPenVPN connect is only available under support.openvpn.net. If you want a localised client, feel free to use one of the open source clients for Mac, Windows, Android. They are all localised.
